lib/node/node.js

```diff
@@ -236,7 +236,7 @@ class Node extends EventEmitter {
    */
 
   error(err) {
-    this.logger.error(err);
+    this.logger.error(err.stack);
```

Comment:
To reviewers: This may be something we should add to `blgr` when NODE_ENV is not set production?

Reply:
Here is an example:

**Before:**
```
[debug] (wallet-http) Request for method=GET path=/wallet/a/account/two (127.0.0.1).
[info] (wallet) Wallet opened (a).
[error] (node) An unexpected error.
```
_It's unclear where the error is originating._

**After:**
```
[debug] (wallet-http) Request for method=GET path=/wallet/a/account/two (127.0.0.1).
[info] (wallet) Wallet opened (a).
[error] (node) Error: An unexpected error.
   at Route.get [as handler] (/home/bcoin/bcoin/lib/wallet/http.js:300:13)
   at Router.handle (/home/bcoin/bcoin/node_modules/bweb/lib/router.js:97:21)
```
_Gives the line number and the file._

Reply:
I like this idea a lot. Something like this would be helpful for finding the exact line of an issue, one that comes to mind in particular is when I was syncing a full node and the disk filled up. I was seeing:

```
[debug] (net) Error: IO error: /path/to/file/.bcoin/chain/034178.ldg: No space left on device (192.175.59.140:8333)
```

It would have been nice to see the stacktrace in the logs

Reply:
Relevant lines of code in `blgr`
- https://github.com/bcoin-org/blgr/blob/e0517108693afa93e4d88206745a98651a2647ef/lib/logger.js#L279
- https://github.com/bcoin-org/blgr/blob/e0517108693afa93e4d88206745a98651a2647ef/lib/logger.js#L503

`logger.logError` already introspects to see if the object is an `Error`

Reply:
Nice, okay. Probably worth a PR there then, I've run into this often enough.

Reply:
Added in https://github.com/bcoin-org/blgr/commit/23406124a11ea8f95fbcc3fca143defea957ac6b

Reply:
This will be resolved when blgr is updated https://github.com/bcoin-org/bcoin/pull/771